config/wdio/wdio.conf.js

```diff
@@ -51,3 +53,28 @@ exports.config = {
     bail: ci,
   },
 };
+
+if (ci) {
+  config.host = 'standalone-chrome';
+}
+
+// This code only executes for monorepos.  It will create a set of suites that can then be executed
+// independently and/or in parallel via 'wdio --suite suite1' for example
+const isRepoTest = !process.cwd().includes('/packages/');
```

Comment:
We need to update this to be `const isRepoTest = !process.cwd().includes('packages');` for windows

Reply:
Good catch